cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/xcm_config.rs

```diff
@@ -215,21 +236,14 @@ impl xcm_executor::Config for XcmConfig {
 				crate::bridge_to_westend_config::BridgeHubWestendChainId,
 				crate::bridge_to_westend_config::AssetHubRococoToAssetHubWestendMessagesLane,
 			>,
-			XcmExportFeeToSibling<
-				bp_rococo::Balance,
-				AccountId,
-				TokenLocation,
-				EthereumNetwork,
-				Self::AssetTransactor,
-				crate::EthereumOutboundQueue,
-			>,
 			SendXcmFeeToAccount<Self::AssetTransactor, TreasuryAccount>,
 		),
 	>;
 	type MessageExporter = (
 		crate::bridge_to_westend_config::ToBridgeHubWestendHaulBlobExporter,
 		crate::bridge_to_bulletin_config::ToRococoBulletinHaulBlobExporter,
 		crate::bridge_to_ethereum_config::SnowbridgeExporter,
+		crate::bridge_to_ethereum_config::SnowbridgeExporterV2,
```

Comment:
Add SnowbridgeExporterV2 side by side with SnowbridgeExporter so to support two versions at the same time.